### PR TITLE
1.1.1 Changes

### PR DESCRIPTION
**Pull request by codebard**
Includes 1.1.1 changes - in sync with 1.1.1 at WP repo

### CODE REVIEW

File: assets/css/admin.css [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+#patreon_attachment_patreon_level_form{
+	display:inline-block;
+	width : 100%;
+	text-align:left;
+	margin: 0px;
+	font-weight: bold;
+}
+#patreon_attachment_patreon_level{
+
```
Comment (spencerhandley) [inline assets/css/admin.css]: Let's make sure spacing on all of this CSS is consistent

File: assets/css/admin.css [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+	font-weight: bold;
+}
+#patreon_attachment_patreon_level{
+	
+	width : auto;
+	padding-left: 3px;
+}
+
+.patreon_image_locking_interface_heading{
+
```
Comment (spencerhandley) [inline assets/css/admin.css]: here

File: assets/css/admin.css [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+	font-weight : bold;
+	margin-left: 15px;
+	display : inline-block;
+}
+
+.thickbox-loading{
+	max-width : 500px !important;
+	width : 100% !important;
+	min-height : 265px !important;
+
```
Comment (spencerhandley) [inline assets/css/admin.css]: here

File: assets/css/admin.css [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+	border-top : 2px solid #dedede;
+	border-bottom : 2px solid #dedede;
+	padding: 15px;
+	-webkit-box-sizing: border-box; /* Safari/Chrome, other WebKit */
+	-moz-box-sizing: border-box;    /* Firefox, other Gecko */
+	box-sizing: border-box;         /* Opera/IE 8+ */
+	color : #737276;
+	font-weight:bold;
+	min-height:150px;
+
```
Comment (spencerhandley) [inline assets/css/admin.css]: here

File: assets/css/admin.css [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+	-webkit-box-sizing: border-box; /* Safari/Chrome, other WebKit */
+	-moz-box-sizing: border-box;    /* Firefox, other Gecko */
+	box-sizing: border-box;         /* Opera/IE 8+ */
+	color : #737276;
+	font-weight:bold;
+	min-height:150px;
+	
+}
+.patreon_image_locking_interface_input_prefix
+{
```
Comment (spencerhandley) [inline assets/css/admin.css]: same line

File: assets/css/admin.css [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+}
+
+.thickbox-loading{
+	max-width : 500px !important;
+	width : 100% !important;
+	min-height : 265px !important;
+	
+	
+}
+.patreon_image_locking_interface_content{
```
Comment (spencerhandley) [inline assets/css/admin.css]: space before bracket

File: assets/css/admin.css [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+	
+}
+.patreon_image_locking_interface_input_prefix
+{
+    background-color: #FFF !important;
+    line-height:20px;
+    height:26px;
+    padding:2px;
+    font-size:14px;
+	color : #000000;
```
Comment (spencerhandley) [inline assets/css/admin.css]: why the extra indent?

File: assets/css/admin.css [hunk 8]
```diff
@@ -0,0 +1,161 @@
...
+	display:inline-block;
+	border : 1px solid #ececec !important;
+	border-top: 2px solid #d3d3d3 !important;
+	padding-left : 10px;
+	width : 100%;
+
+}
+
+.patreon_image_locking_interface_input_prefix INPUT
+{
```
Comment (spencerhandley) [inline assets/css/admin.css]: brackets on same line

File: assets/css/admin.css [hunk 9]
```diff
@@ -0,0 +1,161 @@
...
+    border: 0px !important;
+    background-color: #FFF;
+	height:20px;
```
Comment (spencerhandley) [inline assets/css/admin.css]: indent?

File: assets/css/admin.css [hunk 10]
```diff
@@ -0,0 +1,161 @@
...
+	outline: none !important;
+	box-shadow : none !important;
+	width : 100%;
+}
+
+.patreon-image-locking-update-button{
+
```
Comment (spencerhandley) [inline assets/css/admin.css]: remove spaces

File: assets/css/admin.css [hunk 11]
```diff
@@ -0,0 +1,161 @@
...
+	box-shadow : none !important;
+	width : 100%;
+}
+
+.patreon-image-locking-update-button{
+	
+	float:right;
+	margin : 15px;
+	display:inline-block;
+
```
Comment (spencerhandley) [inline assets/css/admin.css]: remove space

File: assets/css/admin.css [hunk 12]
```diff
@@ -0,0 +1,161 @@
...
+.patreon-image-locking-update-button{
+	
+	float:right;
+	margin : 15px;
+	display:inline-block;
+	
+}
+
+#patreon_attachment_patreon_level_form #TB_closeWindowButton{
+
```
Comment (spencerhandley) [inline assets/css/admin.css]: space

File: assets/css/admin.css [hunk 13]
```diff
@@ -0,0 +1,161 @@
...
+}
+
+#patreon_attachment_patreon_level_form #TB_closeWindowButton{
+	
+	display:inline-block;
+	margin-top:15px;
+	margin-right:15px;
+	width: 20px;
+	height: 20px;
+
```
Comment (spencerhandley) [inline assets/css/admin.css]: space

File: assets/css/admin.css [hunk 14]
```diff
@@ -0,0 +1,161 @@
...
+	width :100% !important;
+	display:inline-block;
+	padding: 15px;
+	padding-top: 0px;
+	-webkit-box-sizing: border-box; /* Safari/Chrome, other WebKit */
+	-moz-box-sizing: border-box;    /* Firefox, other Gecko */
+	box-sizing: border-box;         /* Opera/IE 8+ */
+	color : #737276;
+	font-weight:bold;
+
```
Comment (spencerhandley) [inline assets/css/admin.css]: space

File: assets/css/admin.css [hunk 15]
```diff
@@ -0,0 +1,161 @@
...
+	-webkit-box-sizing: border-box; /* Safari/Chrome, other WebKit */
+	-moz-box-sizing: border-box;    /* Firefox, other Gecko */
+	box-sizing: border-box;         /* Opera/IE 8+ */
+	color : #737276;
+	font-weight:bold;
+	
+}
+
+#patreon-image-toolbar {
+	display:hidden;
```
Comment (spencerhandley) [inline assets/css/admin.css]: indents here

File: assets/css/admin.css [hunk 16]
```diff
@@ -0,0 +1,161 @@
...
+	top : 6px;
+	left : 15px;
+	width:27px;
+	height:41px;
+	padding-top:1px;
+	padding-left: 1px;
+	border: 1px transparent #575c5e;
+}
+#patreon-image-lock-icon:hover {
+
```
Comment (spencerhandley) [inline assets/css/admin.css]: space

File: assets/css/admin.css [hunk 17]
```diff
@@ -0,0 +1,161 @@
...
+	border: 1px solid #575c5e;
+	cursor:pointer;
+	
+}
+#patreon-image-lock-icon img {
+		position:absolute;
+		top : 2px;
+		left : 2px;
+		width :23px;
+		height: 37px;
```
Comment (spencerhandley) [inline assets/css/admin.css]: indents and space

File: assets/css/admin.css (hunk 7 above)
Comment (codebard) [reply assets/css/admin.css]: This indent doesnt exist when viewed in notepad++  - im guessing some display issue at github? If it doesnt display the same when downloaded that is...

**Pull request closed without merging**